crates/ruff_server/src/server/api/requests/code_action.rs

```diff
@@ -147,14 +147,14 @@ fn organize_imports(snapshot: &DocumentSnapshot) -> crate::Result<CodeActionOrCo
             None,
         )
     };
-    let action = types::CodeAction {
+
+    Ok(CodeActionOrCommand::CodeAction(types::CodeAction {
         title: format!("{DIAGNOSTIC_NAME}: Organize imports"),
-        kind: Some(types::CodeActionKind::SOURCE_ORGANIZE_IMPORTS),
-        edit,
-        data,
+        kind: Some(crate::SOURCE_ORGANIZE_IMPORTS_RUFF),
+        edit: edit.clone(),
+        data: data.clone(),
         ..Default::default()
```

Comment:
The clones aren't necessary (weren't needed before)
```suggestion
        edit,
        data,
        ..Default::default()
```